docs/blog/posts/2023/sr-linux-kubernetes-anycast-lab.md

```diff
@@ -0,0 +1,971 @@
+
+At the end of this blog post you can find a [quick summary](#tldr-version) of the steps performed to deploy the lab and configure the use cases.
+
+## Prerequisites
+
+The following tools are required to be installed to run the lab on any Linux host. The links will get you to the installation instructions.
+
+* The lab leverages [Containerlab](https://containerlab.dev/install/) to spin up a Leaf/Spine Fabric coupled with [Minikube](https://minikube.sigs.k8s.io/docs/start/) to deploy the Kubernetes cluster.
+* [Docker engine](https://docs.docker.com/engine/install/) to power containerlab.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) CLI client is also required to interact with the k8s cluster.
+
+## Lab description
+
+### Topology
+
+The goal of this lab is to provide users with an environment to test the network integration of a Kubernetes cluster with a Leaf/Spine SR Linux fabric.
+
+<figure markdown>
+  <div class="mxgraph" style="max-width:100%;border:1px solid transparent;margin:0 auto; display:block;" data-mxgraph='{"page":0,"zoom":1.7,"highlight":"#0000ff","nav":true,"check-visible-state":true,"resize":true,"url":"https://raw.githubusercontent.com/srl-labs/srl-k8s-anycast-lab/main/images/topology.drawio"}'></div>
+  <figcaption>Topology</figcaption>
+</figure>
+
+The setup consists of:
+
+* A Leaf/Spine Fabric: 2xSpines, 4xLeaf switches
+* Minikube kubernetes cluster with MetalLB load balancing implementation (3 nodes)
+* Kubernetes service deployed on top of Cluster
+* Linux clients to simulate connections to k8s service (4 clients)
+
+Courtesy of MetalLB, Kubernetes nodes establish BGP sessions with Leaf switches. Through the BGP sessions the IP addresses of the exposed services (loadBalancerIPs, commonly known as Virtual IPs or VIPs for short) are announced to the IP fabric.
+
+### Kubernetes Service
+
+To illustrate the integration between the workloads running in the k8s cluster and the IP fabric, we will deploy a simple NGINX server replicated across the three k8s nodes. A kubernetes ClusterIP service will be created to expose the NGINX server inside the cluster and a MetalLB loadBalancer service will be created to expose the NGINX server to the outside world.
+
+With simulated clients, we will verify how traffic is distributed among the different nodes/pods using `curl` and reaching over to the exposed service IP address.
+
+### Underlay Networking
+
+The [eBGP unnumbered peering](https://documentation.nokia.com/srlinux/23-3/books/routing-protocols/bgp.html#bgp-unnumbered-peer) makes the core of our IP fabric. Each leaf switch is configured with a unique ASN, whereas all spines share the same ASN, which is a common practice in Leaf/Spine fabrics.
+
+Through eBGP the loopback/system IP addresses are exchanged between the leaves, making it possible to setup iBGP sessions for the overlay services that support the k8s cluster.
```

Comment:
for overlay services we need to add a diagram that shows IBGP peering sessions between the leaves

similar to this (tab [5])
https://app.diagrams.net/?src=about#Uhttps%3A%2F%2Fraw.githubusercontent.com%2Fsrl-labs%2Flearn-srlinux%2Fdiagrams%2Fquickstart.drawio#%7B%22pageId%22%3A%22TaE2ys9rEfVsJn4JcLC4%22%7D

Reply:
added